css/index.css

```diff
@@ -110,6 +161,51 @@ p {
     display: flex;
     align-items: center;
     justify-content: center;
+    animation-name: box-jump;
+    animation-duration: 2s;
+}
+
+
+
+/* Coloring the Boxes */
+.middle-content .boxes .box-1 {
+    background: teal;
+}
+
+.middle-content .boxes .box-2 {
+    background: gold;
+}
+
+.middle-content .boxes .box-3 {
+    background: cadetblue;
+}
+
+.middle-content .boxes .box-4 {
+    background: coral;
+}
+
+.middle-content .boxes .box-5 {
+    background: crimson;
+}
+
+.middle-content .boxes .box-6 {
+    background: forestgreen;
+}
+
+.middle-content .boxes .box-7 {
+    background: darkorchid;
+}
+
+.middle-content .boxes .box-8 {
+    background: hotpink;
+}
+
+.middle-content .boxes .box-9 {
+    background: indigo;
+}
+
+.middle-content .boxes .box-10 {
+    background: dodgerblue;
```

Comment:
https://github.com/MicahJank/Sprint-Challenge--User-Interface/blob/f6044213aaacafacdb293031a2ef32ea8a1706f9/css/index.css#L170-L209

well done here, good use of individual classes